rotkehlchen/tests/utils/database.py

```diff
@@ -59,6 +59,10 @@ def maybe_include_etherscan_key(db: DBHandler, include_etherscan_key: bool) -> N
         service=ExternalService.GNOSIS_ETHERSCAN,
         api_key=ApiKey('J3XEY27VIT7377G34PVPHKWG74NG9PXNSM'),  # Added by dimyG on 2023-10-05
     )])
+    db.add_external_service_credentials([ExternalServiceApiCredentials(
+        service=ExternalService.SCROLL_ETHERSCAN,
+        api_key=ApiKey('THISKEYSHOULDNOTBEUSED'),  # Added by zinkkrysty on 2024-03-05
```

Comment:
add a free scrollscan api key. It's free and easy to do. I can also add one. We just have it for fast tests.

Reply:
Hmm... The ones above didn't work for me, so I assume they were deactivated. I have a free API key but I imagine these can be scraped and abused by bots.

Reply:
They are free. You have to go to scrollscan, create an account and add an api key from there. Each etherscan api is different to the others. Other chain keys won't work.

Reply:
yeah sorry, I edited my answer, I meant they weren't working for the other chains either.

Reply:
addressed in https://github.com/rotki/rotki/pull/7584/commits/a72c9e4587bf541c44eb6f57f50aed3250a07ae2